### PR TITLE
[QUARKS-107] Add use of Range in recipe

### PR DESCRIPTION
**Pull request by dlaboss**

### CODE REVIEW

**Comment by dlaboss** [comment]
@queeniema pls review the changes since you were the recipe's author.  thx!

File: site/recipes/recipe_value_out_of_range.md [hunk 1]
```diff
@@ -137,6 +137,36 @@ When the final application is run, the output looks something like the following
...
 
+## Range values
+
+Filtering against a range of values is such a common analytic activity that the ``quarks.analytics.sensors.Range`` class is provided to assist with that.
+
+Using a Range can simplify and clarify your application code and lessen mistakes that may occur when writing expressions to deal with ranges.
+Though not covered in this recipe, Ranges offer additional conveniences for creating applications with external range specifications and adaptable filters.
+
+In the above examples, a single Range can be used in place of the two
+different expressions for the same logical range:
```
Comment (queeniema) [inline site/recipes/recipe_value_out_of_range.md]: @dlaboss For consistency, could you combine these two lines into one?

**Comment by queeniema** [comment]
These additions look great and simplify the code. Thanks!

File: site/recipes/recipe_value_out_of_range.md (hunk 1 above)
Comment (dlaboss) [inline site/recipes/recipe_value_out_of_range.md]: done!

**Pull request merged**